Makefile

```diff
@@ -678,7 +688,7 @@ remotesystem:
 	if timeout -v 1 true; then \
 		SOCK_FILE=$(shell mktemp --dry-run --tmpdir podman_tmp_XXXX);\
 		export PODMAN_SOCKET=unix://$$SOCK_FILE; \
-		./bin/podman system service --timeout=0 $$PODMAN_SOCKET > $(if $(PODMAN_SERVER_LOG),$(PODMAN_SERVER_LOG),/dev/null) 2>&1 & \
+		./bin/podman system service --timeout=0 --features=testing $$PODMAN_SOCKET > $(if $(PODMAN_SERVER_LOG),$(PODMAN_SERVER_LOG),/dev/null) 2>&1 & \
```

Comment:
I Still see no point in running the system service for this? You could just execute the podman-testing binary now and there really should not need to exists a remote API to corrupt the storage

Reply:
When we're doing remote testing, the client has access to the storage that the server is using?  I thought that we'd taken steps to make that harder in order to avoid cases where we unintentionally used the non-remote code paths.

Reply:
Yes in both e2e and system tests, it is not as straight forward but in both tests suite there are some tests that manually overwrite the default command
https://github.com/containers/podman/blob/e53b96cb25b8210fd4bc6fef743158c67e9809d5/test/system/001-basic.bats#L114
So I don't think there is any problem in just executing the local test binary to corrupt the storage of the service.

Reply:
Okay, then, giving that a try.